mappings/net/minecraft/client/particle/ParticleManager.mapping

```diff
@@ -1,11 +1,17 @@
 CLASS net/minecraft/class_75 net/minecraft/client/particle/ParticleManager
 	FIELD field_269 level Lnet/minecraft/class_18;
+	FIELD field_270 particles [Ljava/util/List;
+		COMMENT Stores a list of lists of particles, indexed by the particle render type.
```

Comment:
apparently indexed is an americanism? (according to one definition on cambridge dictionary for "index-linked"), but it might only be in a legal sense.
Plus I think no british or kiwi person wouldn't use the word "indexed" so this is fine